pkg/controllers/management/nodepool/nodepool.go

```diff
@@ -118,7 +118,10 @@ func (c *Controller) createNode(name string, nodePool *v3.NodePool, simulate boo
 	if simulate {
 		return newNode, nil
 	}
-
+	if delay > time.Duration(0) {
+		time.Sleep(delay)
```

Comment:
This is just doing to delay the create, it doesn't stop the create from happening, if the intentions is to not call create if we don't need to this logic needs to move. However, a create is idempotent.